app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -1,6 +1,21 @@
 class ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher::Runner <
   ManageIQ::Providers::BaseManager::EventCatcher::Runner
-  INCLUDE_EVENTS = {'Hawkular Deployment' => '.*'}.freeze
+
+  def initialize(cfg = {})
+    super(cfg)
```

Comment:
@jshaughn for consistency and avoidance of future failures, I would definitely change this to `super` 
Imagine the args change in the future and someone just greps for the `initialize` of the subclasses and forgets to also check for `super` calls. Then we'd have a :bug: 
So, it's generally best practice to call `super` with no parentheses, unless you really want to change the parameters